components/ActivityForm/index.js

```diff
@@ -12,6 +14,13 @@ export default function ActivityForm({ activity, onSubmit, isUpdateMode }) {
 
   const router = useRouter();
 
+  const countriesOptions = countries.map((country) => {
+    return {
+      value: country.name,
+      label: country.name,
+    };
+  });
```

Comment:
This mapping will happen during each render. That's not necessary. Either move it out of the component below the import statements (then it will be only executed once) or better prepare the data in your assets folder to match this format.